charts/app-config/templates/env-configmap.yaml

```diff
@@ -37,7 +37,8 @@ data:
   RAILS_LOG_TO_STDOUT: "true"
   SENTRY_CURRENT_ENV: {{ .Values.govukEnvironment }}-eks
 
-  PLEK_SERVICE_ASSETS_URI: https://assets.{{ .Values.publishingServiceDomainSuffix }}
+  # TODO: revert assets-eks to assets before launch.
+  PLEK_SERVICE_ASSETS_URI: https://assets-eks.{{ .Values.publishingServiceDomainSuffix }}
   PLEK_SERVICE_DRAFT_ASSETS_URI: https://draft-assets.{{ .Values.publishingServiceDomainSuffix }}
```

Comment:
Should we also be pointing to the draft assets on the test domain?

Reply:
I had the same thought but it looks like we don't have one (yet). Maybe we should? (I guess all it needs is a CNAME and corresponding hostname on the ingress?)

Reply:
Ah yeah - I guess we don't use Fastly for the draft domains. Anyways can sort that out later!